src/main/java/org/terasology/Lakes/LakeDepthFacet.java

```diff
@@ -20,6 +20,9 @@
 import org.terasology.world.generation.Border3D;
 import org.terasology.world.generation.facets.SurfaceHeightFacet;
 
+/**
+ * Marks how deep a lake would be were there some in a particular world xz coordinates.
```

Comment:
Does this facet provide the depth in absolute values or as difference to the "real" surface height?

If I read `LakeDepthProvider` correctly this facet returns a positive floating point value denoting the depth of the lake at a given position:

```suggestion
 * Marks how deep a lake would be at given world coordinates (x, z).
 *
 * The lake depth is given as positive floating point value which should be interpreted as depth from actual surface height.
```